en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -175,6 +173,9 @@ echo $player->dbObject('LastEdited')->Ago();
 The `ORM` uses a "fluent" syntax, where you specify a query by chaining together different methods.  Two common methods
 are `filter()` and `sort()`:
 
+> [!NOTE]
+> Provided `filter` values are automatically escaped and do not require any escaping.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.